src/main/java/com/quiz/Question.java

```diff
@@ -0,0 +1,54 @@
+package com.quiz;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Question {
+
+    private String questionDescription;
+    public List<Answer> questionAnswers;
+
+    public Question(String questionDescription) {
+        this.questionDescription = questionDescription;
+        this.questionAnswers = new ArrayList<>();
+    }
+
+    private Answer findAnswers(String answerDescription) {
```

Comment:
This is a POJO(plain old java object) or model class, if you want functionalities create service for it.